front/package.json

```diff
@@ -15,7 +15,7 @@
     "prebuild": "npm run build:clean",
     "build": "cross-env NODE_ENV=production webpack --config internals/webpack/webpack.config.js --color",
     "build:clean": "rimraf ./build",
-    "build:staging": "cross-env NODE_ENV=staging webpack --config internals/webpack/webpack.config.js --color",
+    "build:staging": "cross-env NODE_ENV=staging --max-old-space-size=4096 webpack --config internals/webpack/webpack.config.js --color",
```

Comment:
Should it be `node --max-old-space-size=4096` ?

Reply:
Yeah, you're right, apparently this did nothing :sweat_smile: 
What's weird is I tried to set it as an env var at the project level and it didn't do the trick, but I'll try it inline...